src/main/resources/database/2021-10-05/09-create-flight.sql

```diff
@@ -0,0 +1,10 @@
+-- liquibase formatted sql
+-- changeset rmecwaldowski:2
+CREATE TABLE flight(
+	id BIGINT AUTO_INCREMENT PRIMARY KEY,
+    type VARCHAR(45) NOT NULL,
+    typeOfSeat VARCHAR(45) NOT NULL,
+    typeOfLuggage VARCHAR(45) NOT NULL,
```

Comment:
I think those 2 attributes were not meant to be in this table.
Instead I'd put departureAirport and arrivalAirport
Of course, we'd need appropriate changes in the `Flight` class as well.